src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/_help.py

```diff
@@ -26,12 +25,12 @@
     type: command
     short-summary: (DEPRECATED) Update the settings of a redis cache.
     long-summary: |
-        WARNING: This command is being deprecated. Please use 'update' command
+        WARNING: This command is deprecated. Instead, use the 'update' command.
```

Comment:
Backticks on update since it references a command.

Reply:
@JasonRShaver - How would you prefer to communicate command deprecation? With lead-ins like this, warnings when running a command, etc?